src/useSimpleAttributeRule.ts

```diff
@@ -0,0 +1,88 @@
+import * as ts from 'typescript';
+import * as Lint from 'tslint';
+
+import {ExtendedMetadata} from './utils/ExtendedMetadata';
+import {getJsxAttributesFromJsxElement} from './utils/JsxAttribute';
+
+export class Rule extends Lint.Rules.AbstractRule {
+
+    public static metadata: ExtendedMetadata = {
```

Comment:
Could you please also add a `rationale`? It's not clear from just the description _why_ we would want such a thing.

Reply:
Well, i tried to look for additional info about that with no success. help will be appreciated.

Reply:
Sure! Something like...

> Simpler attributes in JSX and TSX files helps keep code clean and readable.
> Separate complex expressions into their own line and use clear variable names to make your code more understandable

...maybe?

Reply:
@JoshuaKGoldberg done!
Thanks!